index.js

```diff
@@ -31,16 +31,20 @@ module.exports = {
   },
 
   getOptions() {
+    const hostOptions =
+      (this.parent && this.parent.options) ||
+      (this.app && this.app.options) ||
+      {};
+
     return (
-      (this.app && this.app.options.emberApolloClient) || {
+      hostOptions.emberApolloClient || {
         keepGraphqlFileExtension: true,
       }
     );
   },
 
   setupPreprocessorRegistry(type, registry) {
-    let getOptions = this.getOptions.bind(this);
-    let options = getOptions();
+    const options = this.getOptions();
```

Comment:
This change is not really required to make this PR work, but the usage of `bind` here seemed unnecessary/confusing.